### ISSUE TITLE
Question - Support for kafka 8.2

### ISSUE THREAD

**Issue by spuder** [joekiller/logstash-kafka#54]
The readme only specifically mentions kafka 8.1.1. 
Does this plugin work with kafka 8.2 ?

**Comment by spuder** [comment]
I can confirm that in at least a very basic sense, this plugin does work with 8.2

On the producer

     i hope this works

On the logstash output
```
{
       "message" => "i hope this works",
          "tags" => "_jsonparsefailure",
      "@version" => "1",
    "@timestamp" => "2015-04-03T23:50:40.070Z"
}
```


config
```
input {
  kafka {
    zk_connect => 'zookeeper.example.com:2181/kafka'
    consumer_threads => 3
    topic_id => 'test82'
  }
}
output {
  stdout {
    codec => "rubydebug"
  }
}
```

**Comment by joekiller** [comment]
I recently fixed master for logstash 1.4.X and can confirm the latest will use jruby-kafka 1.X+ which does use Kafka 0.8.2.1 and Scala 2.10

**Comment by spuder** [comment]
Thanks, I'm using logstash 1.5. 
I've been running with kafka 8.2 for a few weeks, things appear to be working well.
